systemtest/src/test/java/io/strimzi/systemtest/rollingupdate/RollingUpdateST.java

```diff
@@ -80,33 +87,50 @@ class RollingUpdateST extends AbstractST {
     private static final Logger LOGGER = LogManager.getLogger(RollingUpdateST.class);
     private static final Pattern ZK_SERVER_STATE = Pattern.compile("zk_server_state\\s+(leader|follower)");
 
+    /**
+     * @description This test case checks recover during Kafka Rolling Update in Zookeeper based Kafka cluster.
+     *
+     * @steps
+     *  1. - Deploy Kafka Cluster with 3 replicas
```

Comment:
```suggestion
     *  1. - Deploy Kafka Cluster with 3 replicas.
```

Reply:
as they are not on other places i did not put it there as well.